WalletWasabi/WabiSabi/Backend/Banning/CoinVerifierApiClient.cs

```diff
@@ -56,13 +54,21 @@ public virtual async Task<ApiResponseItem> SendRequestAsync(Script script, Cance
 		{
 			tries--;
 
+			using CancellationTokenSource timeoutTokenSource = new(ApiRequestTimeout);
+			using CancellationTokenSource linkedTokenSource = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken, timeoutTokenSource.Token);
+
```

Comment:
Fixes a stupid bug. 
When the `100 seconds timeout` elapsed and we retried, we only had 20 seconds to get back a response until the cancellation token cancels the requests.

With this change the CTS has 120 seconds on each try.